### DISCUSSION TITLE
i lost my password ....

### DISCUSSION THREAD

**Discussion by JulaoBR007** [awawa-dev/HyperHDR#572] [Q&A] [answered]
im using webos22.
how to reset password in webos22???

**Comment by awawa-dev** [comment, marked as answer]
there ia s resetPassword  parameter for running HyperHDR app. Or you can delete HyperHDR db configuration file  but HyperHDR must be stopped first and you will lose all HyperHDR settings.

**Reply by JulaoBR007** [reply]
How do I give the reset command, I can't find the executable to run... I can only find the ./hyperhdr folder with the .db file inside but not the executable via ssh

**Reply by awawa-dev** [reply]
I dont know the exactly HyperHDR app path on webos. And probably it's run as some webos service so you need to stop it first. Try ask on the opengltv discord.
PS: third option is to copy .db file to PC/Windows and delete it on webos, unlock it with HyperHDR running on the Windows, next using HyperHDR panel export to json and restore it on the clean HyperHDR on webos.

**Reply by JulaoBR007** [reply]
ok!!! tks!!!
